src/components/Answer.tsx

```diff
@@ -5,7 +5,9 @@ export default function Answer(props: { id: string; background: string; text: st
 
   return (
     <Button onClick={props.setAnswer}
+      _focus={{ background: "lightgrey", color: "black" }}
```

Comment:
This causes a slight bug. When a user clicks on the screen that's not one of the answers - it'll change the button back to whatever color it was before (i.e. if it was green before it'll turn it back to green). This could cause some confusion for users - especially if they are using a mobile device where users pressing randomly on the screen (i.e. on the background) is pretty common. You'll probably need to change the background color and font color on click - and change it back if a user selects another answer. There might be an easier way to do this but I'm not sure - you should probably look into it.

Reply:
As of right now, it looks like a host can "answer" a question since they can click on an answer and it will change color. You'll need to fix this.

Reply:
I changed the implementation of this so it should be good now, It's dependent on the answer state now and not the focus